CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to Open Source Studio at ITP
+
+## Who can contribute?
+**Everyone** is welcome to engage in the open source project that is the organisation you are currently viewing. We encourage discussion and contribution from outside the student body of the class. 
+
+The only limitation as an outside collaborator is the actual class meeting space, a seminar-style meeting in person. This is a private space for the enrolled students to discuss, question, and collaborate on assignments around the practice and philosophy of open source.
+
+## How to contribute
+Please consider our [code of conduct](https://github.com/Open-Source-Studio-at-ITP/Playground/blob/source/CODE_OF_CONDUCT.md) in all your interactions with our project. 
+
+### Substantive changes
+In general, before making a pull request that alters the *substance* of a document (e.g., changing the terms of the licence or adding rules to a code of conduct), discuss the change first by opening an issue, or commenting on an open issue. Possible exceptions include if the change was discussed in class, or if you are Dan.
+
+Use helpful commit messages where possible. For example, "Add diversity statement to Code of Conduct" is more helpful than the default "Update CODE_OF_CONDUCT.md"
+
+### Assignment additions
+Assignment pull requests do not require a new issue, but be sure to identify which assignment you are submitting in the name of your Pull Request (e.g., "Add [Name] week 4 assignment")
```

Comment:
For outside collaborators who'd like to PR assignments for practice and sharing, I suggest considering something like "**Assignment PRs** created by outside collaborators would only be appropriate if the time of the PR creation is during the time of the assignment." 

This might not be a problem for now, and as much as I'd like to share with the class my Blog Post Reflections from week 1 & 2 (which are just not completed yet), but I imagine it would be difficult for maintainers to deal with several "**Week 1 Assignment PRs**" in December, in case this repo gets popular later on.

Reply:
Thanks for the thoughts! My personal take is that it would be a pity to self-censor ideas and thoughts simply because a contributor is late to the party. Of course the person actually managing the repo should have the final say.

Perhaps a way to alleviate the strain you mention is to have a separate folder/file for assignment submissions, in such a way that the main syllabus is separate from the thoughts, ideas, and responses of the assignments?

Reply:
@alfieyfc I am happy to review and merge pull requests for contributions at any time, at this point, the scale of this is small so it's no problem.  I'm not opposed to moving the blog post links from the syllabus document, anyone should feel free to open an issue about this and work on it!